shaman/util.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import datetime
+
+from pecan import conf
+from sqlalchemy import asc
+
+from shaman import models
+
+
+def get_next_node():
+    """
+    Retrieves the next chacra node in
+    the rotation and returns it.
+    """
+    nodes = models.Node.query.filter_by(healthy=True).order_by(
+        asc(models.Node.last_used),
+    )
+    for node in nodes:
+        if is_node_healthy(node):
+            node.last_used = datetime.datetime.now()
```

Comment:
we should use `datetime.utcnow()` so that we are not tied to whatever the system thinks it is `now()`